docs/user_manual/components.md

```diff
@@ -763,16 +763,18 @@ A `fault` has no steady state output.
 
 #### Electric Model
 
+##### Fault types and default values for fault phase
+
 Four types of short circuit fault are included in power-grid-model.
 
-| `fault_type`                       | `fault_phase`    | description                                                            |
-| ---------------------------------- | ---------------- | ---------------------------------------------------------------------- |
-| `FaultType.three_phase`            | `FaultPhase.abc` | Three phases are connected with fault impedance.                       |
-| `FaultType.single_phase_to_ground` | `FaultPhase.a`   | One phase is grounded with fault impedance, and other phases are open. |
-| `FaultType.two_phase`              | `FaultPhase.bc`  | Two phases are connected with fault impedance.                         |
-| `FaultType.two_phase_to_ground`    | `FaultPhase.bc`  | Two phases are connected with fault impedance then grounded.           |
+In case the `fault_phase` is not specified or is equal to `FaultPhase.default_value`, the power-grid-model assumes the fault phases for different values of `fault_type` as specified in the table below.
 
-In case the `fault_phase` is not specified or is equal to `FaultPhase.default_value`, the power-grid-model assumes the following fault phases for different values of `fault_type`.
+| `fault_type`                       | Supported values of `FaultPhase`                  | `FaultPhase.default_value` | description                                                            |
```

Comment:
Why did the content of this table change in this PR?

Reply:
Because the original table did not capture its meaning fully. It's not like `FaultType.two_phase` only supports `FaultPhase.bc`. I can cherry-pick this out of this PR if you prefer

Reply:
BTW: I triggered on this because there was a broken link that attempted to reference this table, but there was nothing it could reference. I switched the order of the table and the text, added a subsection header and slightly modified the content of the table to reflect the goal of the crosslinking

Reply:
If you read the text, that was also not the intention of this table: 

"In case the `fault_phase` is not specified or is equal to `FaultPhase.default_value`, the power-grid-model assumes the following fault phases for different values of `fault_type`"

I think the new table is nice to have, but in that case you should update the text as well ;)

Reply:
No need to cherry-pick this now, since we're already having the discussion

Reply:
OK, will update the text

Reply:
done